### PR TITLE
Update community_process.md

### PR DESCRIPTION
**Pull request by Sayanta66**
Fixed broken link and updated one

### CODE REVIEW

File: site/content/extend/plugins/community_process.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ weight: 50
 ---
 
 
-Getting your plugin onto our Community server https://community.mattermost.com is a valuable source of feedback. Whether you are a [Core Committer](https://github.com/mattermost/mattermost-developer-documentation/blob/dae9ecb3b445111479acbaba9f382e9eb263bc52/contribute/getting-started/core-committers) or anyone from the community, we want you to get feedback to improve your plugin. 
+Getting your plugin onto our Community server https://community.mattermost.com is a valuable source of feedback. Whether you are a [Core Committer](https://developers.mattermost.com/contribute/getting-started/core-committers/) or anyone from the community, we want you to get feedback to improve your plugin.
```
Comment (justinegeffen) [inline site/content/extend/plugins/community_process.md:10]:
```suggestion
Getting your plugin onto our Community server https://community.mattermost.com is a valuable source of feedback. Whether you are a [Core Committer](https://handbook.mattermost.com/contributors/contributors/core-committers) or anyone from the community, we want you to get feedback to improve your plugin.
```

File: site/content/extend/plugins/community_process.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ weight: 50
...
+Getting your plugin onto our Community server https://community.mattermost.com is a valuable source of feedback. Whether you are a [Core Committer](https://developers.mattermost.com/contribute/getting-started/core-committers/) or anyone from the community, we want you to get feedback to improve your plugin. 
 
-However we must ensure that our Community server remains stable for everyone. This document outlines the process of getting your plugin onto the Community server and some of these steps are required to get your plugin into the [Plugin Marketplace](/extend/plugins/community-plugin-marketplace/). 
+However we must ensure that our Community server remains stable for everyone. This document outlines the process of getting your plugin onto the Community server and some of these steps are required to get your plugin into the [Plugin Marketplace](https://mattermost.com/marketplace/).
```
Comment (justinegeffen) [inline site/content/extend/plugins/community_process.md:12]: The existing link points to a page with all the steps needed to follow to list a plugin in the Plugin Marketplace. Having the URL point to the Plugin Marketplace works in some cases but in this case it doesn't. I have reverted this change. Thank you!

Comment (justinegeffen) [inline site/content/extend/plugins/community_process.md:12]:
```suggestion
However we must ensure that our Community server remains stable for everyone. This document outlines the process of getting your plugin onto the Community server and some of these steps are required to get your plugin into the [Plugin Marketplace](/extend/plugins/community-plugin-marketplace/).
```

File: site/content/extend/plugins/community_process.md [hunk 3]
```diff
@@ -7,9 +7,9 @@ weight: 50
 ---
 
 
-Getting your plugin onto our Community server https://community.mattermost.com is a valuable source of feedback. Whether you are a [Core Committer](https://github.com/mattermost/mattermost-developer-documentation/blob/dae9ecb3b445111479acbaba9f382e9eb263bc52/contribute/getting-started/core-committers) or anyone from the community, we want you to get feedback to improve your plugin. 
+Getting your plugin onto our Community server https://community.mattermost.com is a valuable source of feedback. Whether you are a [Core Committer](https://handbook.mattermost.com/contributors/contributors/core-committers) or anyone from the community, we want you to get feedback to improve your plugin.
```
Comment (cwarnermm) [inline site/content/extend/plugins/community_process.md:10]:
```suggestion
Getting your plugin onto our [Community server](https://community.mattermost.com) is a valuable source of feedback. Whether you are a [Core Committer](https://handbook.mattermost.com/contributors/contributors/core-committers) or anyone from the community, we want you to get feedback to improve your plugin.
```

File: site/content/extend/plugins/community_process.md [hunk 4]
```diff
@@ -7,9 +7,9 @@ weight: 50
...
 
-However we must ensure that our Community server remains stable for everyone. This document outlines the process of getting your plugin onto the Community server and some of these steps are required to get your plugin into the [Plugin Marketplace](/extend/plugins/community-plugin-marketplace/). 
+However we must ensure that our Community server remains stable for everyone. This document outlines the process of getting your plugin onto the Community server and some of these steps are required to get your plugin into the [Plugin Marketplace](/extend/plugins/community-plugin-marketplace/).
```
Comment (cwarnermm) [inline site/content/extend/plugins/community_process.md:12]:
```suggestion
However we must ensure that our Community server remains stable for everyone. This document outlines the process of getting your plugin onto the Community server. Some of these steps are also required to get your plugin into the [Plugin Marketplace](/extend/plugins/community-plugin-marketplace/).
```

**Review by cwarnermm — CHANGES_REQUESTED** [review]
Thank you for your docs contribution! I've provided feedback inline.

**Comment by justinegeffen** [comment]
This page was updated as part of a separate PR and was moved to another section which is why there are conflicts. As this PR is not going to be applied, I am closing it but not marking it as invalid because it was an inadvertent conflict.

**Pull request closed without merging**